tests/test_api.py

```diff
@@ -129,17 +129,39 @@ def test_cb(self):
         self.assertEqual(resp.status_code, 500)  # TODO fix
 
     def test_evaluate_policy(self):
-        kwargs = dict()
-        json_request_body  = {
-            "Action": "dss:CreateSubscription",
-            "Resource": "arn:hca:dss:*:*:subscriptions/${user_id}/*",
-            "Principle": "test@email.com"
-        }
-        kwargs['data'] = json.dumps(json_request_body)
-        kwargs['headers'] = {}
-        kwargs['headers']['Content-Type'] = "application/json"
-        resp = self.app.get('/policies/evaluate', kwargs)
-        self.assertEqual(resp.status_code, 405)  # TODO fix
+        email = "test@email.com"
+        tests = [
+            {
+                'json_request_body': {
+                    "action": "dss:CreateSubscription",
+                    "resource": f"arn:hca:dss:*:*:subscriptions/{email}/*",
+                    "principal": "test@email.com"
+                },
+                'response': {
+                    'code': 200,
+                    'result': ['implicitDeny']
```

Comment:
If something is `implicitDeny` would that mean that it does not have access for the action? I would think that having the 403 would be desirable to indicate that the action is forbidden and not showing information back in the response about what is allowed for a user.  I see that the yaml file (line 51 was also adjusted to reflect [this](https://github.com/HumanCellAtlas/fusillade/pull/41/files)

Reply:
A 403 would be returned if the client did not have permission to hit the `POST /policies/evaluate`  endpoint. Returning a 200 tells the client that policies were successfully evaluated and this is the result.

Reply:
A use case would be a user asks DSS for something. The DSS sends a request to fusillade/policies/evaluate to see if the user has permission. Fusillade returns True or False and the DSS return the resource or 403 depending on the response from fusillade.

Reply:
That makes sense, so it would be up to the service utilizing Fusillade to manage this and respond appropriately. Thanks for clarifying.